.github/actions/abort-release-candidate-v1/action.yml

```diff
@@ -0,0 +1,106 @@
+name: abort-release-candidate
+description: 'A GitHub Action to abort a release candidate'
+
+inputs:
+  token:
+    description: 'Token used for the GH CLI to close and move issue(s) to the correct column. See README for required permissions'
+    required: true
+  base:
+    description: 'The branch the release candidate will be merged into'
+    required: true
+  docs-repo:
+    description: 'The docs repository where the issue for the release candidate is located'
+    default: 'null'
+    required: false
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+      with:
```

Comment:
Any reason to not use `ref` here and checkout `release`? Then you don't need a separate step to do the branch checkout. https://github.com/actions/checkout?tab=readme-ov-file#usage

Reply:
No particular reason - can update to use `ref` to look at `release`!